src/models/eventsFactory.js

```diff
@@ -172,6 +172,29 @@ class EventsFactory extends Factory {
     return result;
   }
 
+  /**
+   * Returns events that occurred after this timestamp
+   *
+   * @param {Number} limit - events count limitations
+   * @param {Number} skip - certain number of documents to skip
+   * @return {DailyEventInfo[]}
+   */
+  async findChartData(minTimestamp = 1586545200) {
```

Comment:
Этот метод по сути возвращает daily events. Во-первых, такой возможно уже есть, во-вторых, если нет, то лучше поменять сигнатуру на
```suggestion
  async getDailyEvents(since = 1586545200) {
```

И почему тут захардкожено дефолтное значение?